donation_base/security/tax_receipt_security.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  © 2014-2016 Barroux Abbey (http://www.barroux.org)
+  © 2014-2016 Akretion France (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo noupdate="1">
+    <record id="donation_tax_receipt_company_rule" model="ir.rule">
+        <field name="name">Donation Tax Receipt Multi-company</field>
+        <field name="model_id" ref="model_donation_tax_receipt" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
```

Comment:
why was this changed from child_of to in?

Reply:
Companies on 13 have no parent relation

Reply:
yes, there is
https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/res_company.py#L68

Reply:
But there is no parent path, so it shouldn't work

Reply:
Also, you can check this on the migration guide

Reply:
it does also work without parent_path
https://github.com/odoo/odoo/blob/14.0/odoo/osv/expression.py#L526

and there is a difference between in, only including the actual ids, and child_of, actual_ids and their childs. so this would change this function.
not sure whats the authors original intend was, but in migration I would like to avoid a functional change

Reply:
In any case, the standard usage in 13 is to use in company_ids, so I think it is not an issue

Reply:
it wasn't an issue for the beginning, so why change it?

Reply:
Because it is the new behavior, company_ids will take the company that the user is seeing right now. So the right option is to use it this way. So, you might have selected to view 3 companies and, with the new logic, they will not be related. You must be able to see all the information at the same time.

The parent field is hidden and is no longer necessary to use it with this changes. All the logic on 13 (in Odoo and OCA) follows this behavior.

Reply:
I confirm that the change is good.